.changeset/afraid-apricots-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+The hook `astro:config:setup` now provide a `"sync"` variant for the `command` property. This value is usually set when calling the command `astro sync`
```

Comment:
```suggestion
Adds a new variant `sync` for the `astro:config:setup` hook's `command` property. Set this value to call the command `astro sync`.
```
I wasn't sure whether this was something that the integration author configures, or whether it "gets" set by Astro on them, but if the integration author sets this, I might phrase it more about what they need to do.

And "usually" in docs makes me nervous! :sweat_smile: Is there a particular concern/case where this wouldn't happen/work? Or do you just mean that this is "common usage" (in which case, I don't think it's necessary to say usually)

Reply:
Astro sets it so that integration authors know their integration run while a user executes `astro sync`

Reply:
Thanks for the clarification, Florian! Then the existing wording is great, but *usually* still makes me nervous? Would you say that's a key word to include here?

Reply:
No it's confusing, lets remove it

Reply:
OK, I updated my suggestion above taking all this into account!